templates/login.hbs

```diff
@@ -1,4 +1,17 @@
 <form action="{{ action }}" method="post">
+
+  {{#if logoUrl}}
+    <!-- show app logo -->
+    <fieldset><img src={{logoUrl}} style="max-width: 100%"/></fieldset>
+    {{#else}}
+    
+    <!-- show curveball logo -->
+    <fieldset style="background: #1e88e5;">
+      <img src="https://curveballjs.org/assets/images/logo.svg" style="max-width: 100%"/>
+    </fieldset>
+    {{/else}}
+  {{/if}}
```

Comment:
* `fieldset` is a bit strange, no?
* Doesn't need the `else` statement, if the 'default logo' is provided by the settings instead.
* Could the logo instead be added to the 'parent' template, so it shows up on registration, login, lost password, etc?